PROJECT_IDEA_TEMPLATE.md

```diff
@@ -7,7 +7,7 @@
 - Description:
 - Expected Outcome:
 - Recommended Skills:
-- Mentor(s):
+- Mentor(s): It is recommended to have at least 2 mentors, and at least one of them should be the primary mentor. For GSoC, it is **required** to have at least 2 mentors.
```

Comment:
We're contradicting here one sentence to the next. We're suggesting that this become a requirement moving forward.

Reply:
I thought this update was specifically for GSoC, not all the programs generally.

Reply:
```suggestion
- Mentor(s): # It is recommended to have at least 2 mentors, and at least one of them should be the primary mentor. For GSoC, it is **required** to have at least 2 mentors.
  - Jane Doe (@jane-github) - primary
  - John Doe (@john-github)
```